.templaterc.json

```diff
@@ -0,0 +1,20 @@
+{
+    "files": [
+        ".github/*",
+        "book/img/*",
+        "book/preliminary/*",
+        "book/projects/project_initialization.md",
+        "book/projects/project_roadmap.md",
+        "book/projects/project_roles.md",
+        "book/CoC.md",
+        "book/_config.md",
+        "book/_toc.md",
+        "book/intro.md",
+        "book/mission.md",
+        "scripts/*",
+        "{{ cookiecutter.repo_directory }}/*",
+        "CHANGELOG.md",
+        "LICENSE.md",
+        "README.md"
+    ]
```

Comment:
Add:
-  `gitignore`
- 'edtiorconfig`
- `cookiecutter.yaml`
- 'conda` folder

Reply:
I added `gitignore` and `editorconfig`, but I'd intentionally left out the `cookiecutter.yaml` and `conda` dir because they will be updated for individual uses and we don't want to just overwrite them. They're in the category of files that sparked my Slack post about which things to auto-update or not.

Reply:
So you would manually transfer over any new packages that are required to build the website if there are new dependeices?

Reply:
> So you would manually transfer over any new packages that are required to build the website if there are new dependencies?

That sounds awful - so ultimately no. But otherwise the updates will overwrite files such that any packages added to the environment for an repo that used the template will be perpetually removed.

Reply:
Looks to me that we have a need to keep the 'tutorial' libraries in a separate file. Conda can [use multiple files to create an environment](https://docs.conda.io/projects/conda/en/latest/commands/create.html#Named%20Arguments) and sounds a candidate to consider.
The cookiecutter also lists dependent extensions. Maybe that can be outsourced to a dedicated file too

Log issue?

Reply:
I added a [more specific list](https://github.com/uwhackweek/jupyterbook-template/issues/68#issuecomment-1017756477) that includes these items to my update in the existing issue #68

Reply:
Reading through the issue, I think the `conda/lock_environment.sh` should be added to this list.